messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/jms/ExternalJMSQueueDefinition.java

```diff
@@ -51,7 +52,11 @@ private ExternalJMSQueueDefinition() {
 
     @Override
     public void registerAttributes(ManagementResourceRegistration registry) {
-        registry.registerReadWriteAttribute(CommonAttributes.DESTINATION_ENTRIES, null, new ReloadRequiredWriteAttributeHandler(CommonAttributes.DESTINATION_ENTRIES));
+        if (registerRuntimeOnly) {
+            registry.registerReadOnlyAttribute(CommonAttributes.DESTINATION_ENTRIES, null);
```

Comment:
change is fine (JMS resources from deployment are read-only) but how is it related to the connector/acceptor capabilities change?

Reply:
I detected the issue (aka missing runtime description) while fixing this. Maybe i could split the commit in 2